dreadnode/main.py

```diff
@@ -278,7 +426,7 @@ def configure(
         if self.server or self.token or self.local_dir:
             destination = self.server or DEFAULT_SERVER_URL or "local storage"
             logging_console.print(
-                f"Dreadnode logging to [orange_red1]{destination}[/] ({config_source}, project: {self.project or 'default'})"
+                f"Dreadnode logging to [orange_red1]{destination}[/] ({config_source})"
```

Comment:
If possible, I think it would be good to unify the logging statement logic so we get a unified message about server, workspace, project, profile, etc.